test/pom.xml

```diff
@@ -182,6 +182,14 @@
       <artifactId>dropwizard-http2</artifactId>
     </dependency>
 
+    <!-- Dropwizard 1.1.x requires Mockito as a test dependency -->
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
+      <version>2.7.6</version>
```

Comment:
This version should be able to be omitted, but if not we should add it in the `dependenciesManagement` section of the parent-pom and omit the version here.